### PR TITLE
integration tests

### PR DESCRIPTION
**Pull request by kauanslr**

### CODE REVIEW

File: tests/Integration/Publisher/PublisherTest.php [hunk 1]
```diff
@@ -66,6 +66,8 @@ public function test_it_should_publish_a_message_using_routing()
         $this->pigeon->routing($this->routing_key)
             ->publish($data);
 
+        sleep(1);
```
Comment (Arkanius) [inline tests/Integration/Publisher/PublisherTest.php]: ?

Comment (kauanslr) [reply tests/Integration/Publisher/PublisherTest.php]: wait message go to rabbit driver...

**Comment by Arkanius** [comment]
@kauanslr fix conflicts please

**Comment by kauanslr** [comment]
> @kauanslr fix conflicts please

Fixed

**Pull request merged**